### PR TITLE
[Feature][AES-1818] Adjust styles within TwoColumnList component

### PR DESCRIPTION
**Pull request by beau-charman**
This PR adjusts styles within the TwoColumnList component, such as font weights, colours and sizes, to address https://aesoponline.atlassian.net/browse/AES-1818

### CODE REVIEW

File: src/components/TwoColumnList/TwoColumnList.module.css [hunk 1]
```diff
@@ -1,14 +1,9 @@
...
-
-  * {
-    &,
-    &::before,
-    &::after {
-      box-sizing: inherit;
-    }
+  &,
+  & * {
+    box-sizing: border-box;
```
Comment (hassanraza0054) [inline src/components/TwoColumnList/TwoColumnList.module.css:6]: A bit unfamiliar with this but could you tell what's the difference b/w **&** and **& *** in this context and what the stylings aim to achieve while being nested in base? Thank you!

Comment (beau-charman) [reply src/components/TwoColumnList/TwoColumnList.module.css:6]: `&` represents the current selector context, and `*` is a wildcard selector (so everything), and the space denotes children, so basically this outputs: 

```
.base, .base * { ... }
```

As I want to apply the style to both the current selector and all it's children, but I don't want to duplicate the declaration

File: src/components/TwoColumnList/TwoColumnList.module.css [hunk 2]
```diff
@@ -17,87 +12,67 @@
   }
 }
```
Comment (hassanraza0054) [inline src/components/TwoColumnList/TwoColumnList.module.css:13]: image

Was going over the CSS and couldn't find the side padding required. Perhaps it was missed? Just want to be sure so dropping a comment!

**Review by hassanraza0054 — CHANGES_REQUESTED** [review]

File: src/components/TwoColumnList/TwoColumnList.js [hunk 3]
```diff
@@ -37,22 +38,32 @@ export const Column = ({ lists, theme }) =>
     return (
       <div className={styles.wrapper} key={id}>
         {heading && (
-          <Heading level="2" size="medium" theme={theme}>
```
Comment (hassanraza0054) [inline src/components/TwoColumnList/TwoColumnList.js:40]: Just wanted to confirm why the original size was medium and we proceeded to large. See no particular issue but just curious.

File: src/components/TwoColumnList/TwoColumnList.module.css (hunk 2 above)
Comment (beau-charman) [reply src/components/TwoColumnList/TwoColumnList.module.css:13]: This is applied in the `TwoColumnLayout` component

File: src/components/TwoColumnList/TwoColumnList.js (hunk 3 above)
Comment (beau-charman) [reply src/components/TwoColumnList/TwoColumnList.js:40]: It better reflects the requirements in the ticket

File: src/components/TwoColumnList/TwoColumnList.module.css (hunk 2 above)
Comment (beau-charman) [reply src/components/TwoColumnList/TwoColumnList.module.css:13]: src/components/TwoColumnLayout/TwoColumnLayout.module.css

Comment (hassanraza0054) [reply src/components/TwoColumnList/TwoColumnList.module.css:13]: Awesome just saw it. Ty Beau!

**Review by hassanraza0054 — APPROVED** [review]

**Pull request merged**